en/concept/system_startup.md

```diff
@@ -68,34 +68,49 @@ The resulting boot log has detailed information about the boot sequence and shou
 
 #### Common boot failure causes
 
-  * For custom applications: The system was out of RAM. Run the `free` command to see the amount of free RAM.
-  * A software fault or assertion resulting in a stack trace
+- For custom applications: The system was out of RAM. Run the `free` command to see the amount of free RAM.
+- A software fault or assertion resulting in a stack trace
 
 ### Replacing the System Startup
 
-In most cases customizing the default boot is the better approach, which is documented below. If the complete boot should be replaced, create a file `/fs/microsd/etc/rc.txt`, which is located in the `etc` folder on the microSD card. If this file is present nothing in the system will be auto-started.
+In most cases customizing the default boot is the better approach, which is documented below.
+If the complete boot should be replaced, create a file `/fs/microsd/etc/rc.txt`, which is located in the `etc` folder on the microSD card.
+If this file is present nothing in the system will be auto-started.
```

Comment:
How then do you start stuff? Is the idea that you put all your config in this file?

Reply:
Thanks. All fixed in https://github.com/PX4/PX4-user_guide/pull/2071/files#r988606065